builtin/update-index.c

```diff
@@ -1098,7 +1099,13 @@ int cmd_update_index(int argc, const char **argv, const char *prefix)
 	argc = parse_options_end(&ctx);
 
 	getline_fn = nul_term_line ? strbuf_getline_nul : strbuf_getline_lf;
+	if (mark_skip_worktree_only && gvfs_config_is_set(-1))
+		die("Modifying the skip worktree bit is not supported on a GVFS repo");
```

Comment:
Please start `die()` messages with a lower-case letter (this is a convention that Junio *really* cares about), and if we ever want to upstream this, we will have to mark it for translation (by enclosing the string in `_(...)`).

Reply:
Thanks - I'll do that.  I copy/pasted the old errors out of the GVFS code but can munge them to make Junio happy. :)